packages/jest-matcher-utils/src/deepCyclicCopyReplaceable.ts

```diff
@@ -42,6 +44,9 @@ export default function deepCyclicCopyReplaceable<T>(
     return deepCyclicCopyMap(value, cycles);
   } else if (isBuiltInObject(value)) {
     return value;
+  } else if (plugins.DOMElement.test(value)) {
+    //@ts-expect-error skip casting to Node
+    return value.cloneNode(true);
```

Comment:
```suggestion
    return (((value as unknown) as Element).cloneNode(true) as unknown) as T;
```
Can fix error with this, but a little bit ugly.

Reply:
Comment override is fine